app/elements/youneeq-tracker/youneeq-tracker.js

```diff
@@ -134,7 +134,7 @@ class youneeqTracker {
   }
 
   _contentChanged(content, oldContent) {
-    if (typeof content !== 'undefined' && Object.keys(content).length > 0) {
+    if (typeof content !== 'undefined' && Object.keys(content).length > 0 && !this.hidden) {
```

Comment:
Is adding the hidden check needed here and below? This element should never be hidden as it sits at the global level.